src/main/resources/assets/carpet/lang/fr_fr.json

```diff
@@ -0,0 +1,36 @@
+{
+  // TODO Rules
+  
+  // Categories:
+
+  "carpet.category.bugfix": "correction de bugs",
+  "carpet.category.survival": "survie",
+  "carpet.category.creative": "créatif",
+  "carpet.category.experimental": "expérimental",
+  "carpet.category.optimization": "optimisation",
+  "carpet.category.feature": "fonctionnalité",
+  "carpet.category.command": "commande",
+  "carpet.category.tnt": "TNT",
+  "carpet.category.dispenser": "distributeur",
+  "carpet.category.scarpet": "tapis",
```

Comment:
This should probably just be `scarpet`, since it's referring to the programming language itself, not to a carpet.